synapse/handlers/room.py

```diff
@@ -959,7 +967,10 @@ async def create_room(
                     ratelimit=False,
                     content=content,
                     new_room=True,
+                    prev_event_ids=[last_sent_event_id],
+                    depth=depth,
                 )
+                depth += 1
```

Comment:
We're not updating the `last_send_event_id` here, so I think this creates a forky DAG rather than a nice linear line.

Reply:
Whoops! Fixed in https://github.com/matrix-org/synapse/pull/13224/commits/98414efca28178c87afa71c0e58cc28e27cf82a8